src/lua/alloc.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2010-2024, Tarantool AUTHORS, please see AUTHORS file.
```

Comment:
Let's use `SPDX-License-Identifier` (grep for examples).

Reply:
Done. Thank you for that idea.

Reply:
I would do it in the header file too.

Reply:
Oh, I am sorry. Applied